5.5/root/usr/share/container-scripts/mysql/lib.sh

```diff
@@ -0,0 +1,3 @@
+function info-log {
```

Comment:
let's not dwell on this.  lib.sh is fine.  my "morecommon.sh" suggestion was a joke, if that wasn't clear.